assembl/static2/js/app/components/common/footer.jsx

```diff
@@ -39,18 +45,18 @@ class Footer extends React.Component {
               </Link>
             </div>
             <div className="terms">
-              {termsOfUseUrl && (
+              {termsAndConditions && (
                 <div className="terms-of-use">
-                  <Link to={termsOfUseUrl} target="_blank">
+                  <Link to={`${get('terms', slug)}`}>
                     <Translate value="footer.terms" />
                   </Link>
                 </div>
               )}
-              {termsOfUseUrl && legalNoticeUrl && <span className="small-hyphen-padding"> &mdash; </span>}
-              {legalNoticeUrl && (
-                <div className="legal-notices">
-                  <Link to={legalNoticeUrl} target="_blank">
-                    <Translate value="footer.legalNotices" />
+              {termsAndConditions && legalNotice && <span className="small-hyphen-padding"> &mdash; </span>}
```

Comment:
are you sure that we don't want legal notice if they are not empty but the `termsAndConditions` are ? it seems that it was the case before but I don't understand why